.ci/build.sh

```diff
@@ -10,17 +10,17 @@ docker login -u ${DOCKER_HUB_LOGIN} -p ${DOCKER_HUB_PASSWORD}
 # Run tests and all the other checks on repo
 docker build -f ${ROOT_DIR}/build/CI/Dockerfile ${ROOT_DIR}
 
-# Unified Plugin Broker
-docker build -t eclipse/che-unified-plugin-broker:latest -f ${ROOT_DIR}/build/unified/Dockerfile ${ROOT_DIR}
-docker tag eclipse/che-unified-plugin-broker:latest eclipse/che-unified-plugin-broker:${TAG}
-docker push eclipse/che-unified-plugin-broker:latest
-docker push eclipse/che-unified-plugin-broker:${TAG}
+# Metadata Plugin Broker
+docker build -t eclipse/che-metadata-plugin-broker:latest -f ${ROOT_DIR}/build/metadata/Dockerfile ${ROOT_DIR}
```

Comment:
Previously we had like Theia Plugin, VsCode Plugin and Theia Plugin Broker, VsCode Plugin Broker. Maybe it would be better wording if we have che-plugin-metadata-broker since it brokers Plugin Metadata but not Metadata Plugin.
I may be wrong here and just wanted to put additional attention here.

P.S. The same about plugin artifacts broker

Reply:
I think around the time of merging the brokers into the unified one, we switched to using `eclipse/che-unified-plugin-broker`, so I chose the names to match that. You suggestion makes more sense, though -- I'll update the relevant PRs.